pylib/Makefile

```diff
@@ -111,3 +111,9 @@ develop: $(BUILD_STEPS)
 anki/buildinfo.py: ../meta/version ../meta/buildhash
 	echo "buildhash='$$(cat ../meta/buildhash)'" > $@
 	echo "version='$$(cat ../meta/version)'" >> $@
+
+VER := $(shell cat ../meta/version)
+.build/vernum: ../meta/version
+	sed -i.bak 's/.*automatically updated.*/    version="$(VER)",  # automatically updated/' setup.py
+	rm setup.py.bak
+	@touch $@
```

Comment:
This looks like a change that was required for the sdist work you were doing, and should not be needed at the moment?

Reply:
You are right. If this is only required if you try to build anki as sdist instead of bdist_wheel. But having this allow me to use `sdist` too and cargo already requires this make rule to update its version. So it is just something you have to do anyway (for cargo.toml).